haskell-src/Concordium/Wasm.hs

```diff
@@ -568,7 +568,7 @@ data SenderPolicy = SenderPolicy {
   spItems :: ![(AttributeTag, AttributeValue)]
   }
 
-mkSenderPolicy :: AccountCredential -> SenderPolicy
+mkSenderPolicy :: AccountCredential' credTy -> SenderPolicy
```

Comment:
Document `credTy`.
```suggestion
-- The type parameter @credTy@ determines the representation
-- of credential registration IDs.
mkSenderPolicy :: AccountCredential' credTy -> SenderPolicy
```

Reply:
THis is documented as part of the `AccountCredential` type already. I do not think it is necessary or practical to also document it at every occurrence.